package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "v8-to-istanbul",
-  "version": "8.1.0",
+  "version": "8.1.1",
```

```suggestion
  "version": "8.1.0",
```

this is updated when publishing

---

ok, I will remove it.